{"name": "Успешная авторизация под существующим пользователем", "status": "passed", "description": "Проверка логина под существующим пользователем", "start": 1739821177104, "stop": 1739821177290, "uuid": "58547032-8da9-4bf6-9263-9a2371434f68", "historyId": "cdd1991503565ca8a8d2101976b3f5b2", "testCaseId": "cdd1991503565ca8a8d2101976b3f5b2", "fullName": "tests.login_user_test.TestLoginUser#test_login_existing_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "login_user_test"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "WIN-E39S914M1UH"}, {"name": "thread", "value": "15872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.login_user_test"}]}